### PR TITLE
Author reward dest pallet benchmark

### PR DESCRIPTION
**Pull request by gui1117**
Simple benchmark for pallet author reward dest, without the final benchmarked weight.

### CODE REVIEW

**Comment by shawntabrizi** [comment]
`cargo test -p pallet-author-reward-dest --features runtime-benchmarks` is working, so thats a good start.

**Comment by shawntabrizi** [comment]
@thiolliere should be fixed now. I ran your exact commands above successfully.

You could also change to use `--chain=main` rather than building a spec file

File: pallets/author-reward-dest/src/weights.rs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	/// Storage: `AuthorRewardDest::Author` (r:1 w:1)
+	/// Proof: `AuthorRewardDest::Author` (`max_values`: Some(1), `max_size`: Some(32), added: 527, mode: `MaxEncodedLen`)
+	fn set_author_reward_dest() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `42`
+		//  Estimated: `1517`
+		// Minimum execution time: 4_000_000 picoseconds.
+		Weight::from_parts(5_000_000, 1517)
+			.saturating_add(RocksDbWeight::get().reads(1_u64))
+			.saturating_add(RocksDbWeight::get().writes(1_u64))
```
Comment (shawntabrizi) [inline pallets/author-reward-dest/src/weights.rs:49]: technically, you delete the storage at the end of the block, so you don't even need to count the storage write, but its okay, better to be conservative

**Review by shawntabrizi — APPROVED** [review]

File: pallets/author-reward-dest/src/weights.rs (hunk 1 above)
Comment (gui1117) [reply pallets/author-reward-dest/src/weights.rs:49]: I see, I should have put the `finalize` hook in the benchmark

**Pull request merged**